### PR TITLE
Clean phone numbers in application

### PR DESCRIPTION
**Pull request by GauravRanganath**
## Overview

- Resolves IEEE-59
- Removed all non numeric characters before saving to database

## Steps to QA

- Try submitting an application with a phone number that has non-numeric characters (ex: + 1 123-456 7890) and check that non-numeric characters are removed in the database

### CODE REVIEW

File: hackathon_site/registration/forms.py [hunk 1]
```diff
@@ -11,6 +11,8 @@
 from registration.models import Application, Team, User
 from registration.widgets import MaterialFileInput
 
+import re
```
Comment (grahamhoyes) [inline hackathon_site/registration/forms.py]: Can you move this up to below the `dateutil` import, with the rest of the stdlib imports?

File: hackathon_site/registration/forms.py [hunk 2]
```diff
@@ -171,6 +173,7 @@ def save(self, commit=True):
 
         self.instance.user = self.user
         self.instance.team = team
+        self.instance.phone_number = re.sub("[^0-9]", "", self.instance.phone_number)
```
Comment (grahamhoyes) [inline hackathon_site/registration/forms.py:175]: Needs a unit test, you can put it in the sign up form tests

Comment (GauravRanganath) [reply hackathon_site/registration/forms.py:175]: I think it makes more sense to put it in the application form tests no?

Comment (grahamhoyes) [reply hackathon_site/registration/forms.py:175]: Whoops, yes you're right!

File: hackathon_site/registration/forms.py (hunk 1 above)
Comment (grahamhoyes) [reply hackathon_site/registration/forms.py]: Still need to move this to below the `dateutil` import to keep our imports organized

Comment (grahamhoyes) [reply hackathon_site/registration/forms.py]: Still needs to be moved up where it should be

**Review by grahamhoyes — APPROVED** [review]

**Pull request merged**